website/docs/language/values/variables.mdx

```diff
@@ -128,7 +128,7 @@ collections:
 The keyword `any` may be used to indicate that any type is acceptable. For
 more information on the meaning and behavior of these different types, as well
 as detailed information about automatic conversion of complex types, see
```

Comment:
```suggestion
The keyword `any` may indicate that any type is acceptable. For
more information on the meaning and behavior of these different types, as well
as detailed information about automatic conversion of complex types, refer to
```

Suggest changing "may be used to indicate" to "may indicate" to avoid passive voice. Also suggest changing "see" to "refer to" to avoid ableist language.